src/vs/workbench/contrib/chat/browser/chatListRenderer.ts

```diff
@@ -417,7 +416,7 @@ export class ChatListItemRenderer extends Disposable implements ITreeRenderer<Ch
 				const plusCursor = (renderValue.value.match(/```\s*$/) ?
 					renderValue.value + '\n\n' :
 					renderValue.value) + ` ${ChatListItemRenderer.cursorCharacter}`;
-				const result = this.renderMarkdown(new MarkdownString(plusCursor), element, disposables, templateData, true);
+				const result = this.renderMarkdown(new MarkdownString(plusCursor, { supportThemeIcons: true }), element, disposables, templateData, true);
```

Comment:
I'm fine rendering these for the placeholder in a progress update, but I don't want to enable this for all content coming out of the LLM, because it doesn't know that it is being interpreted as this special language.

Reply:
Yeah, reworked in https://github.com/microsoft/vscode/pull/190782/